config/software/makedepend.rb

```diff
@@ -19,7 +19,7 @@
 default_version "1.0.5"
 
 source url: "http://xorg.freedesktop.org/releases/individual/util/makedepend-1.0.5.tar.gz",
```

Comment:
Would it be possible to switch to HTTPS?

(several other occurrences, I'd recommend doing a global search across  the repo for `http` without a `s`)

Reply:
```suggestion
source url: "https://xorg.freedesktop.org/releases/individual/util/makedepend-1.0.5.tar.gz",
```